ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -61,6 +61,16 @@ If.speaker {
 	]
 }
 
+If.speakersw {
+        Condition {
+                Type ControlExists
+                Control "name='Master Playback Switch'"
```

Comment:
it's a bit inconsistent with the unconditional use of 'Master Playback Switch' line 50

Reply:
does it look better if the sysw move to line 50 and remove If.speakersw block?
```
If.master {                                                                      
        Condition {                                                              
                Type ControlExists                                               
                Control "name='Master Playback Volume'"                          
        }                                                                        
        True.BootSequence [                                                      
                cset "name='Master Playback Volume' 60%"                         
                cset "name='Master Playback Switch' on"                          
                sysw "-/class/sound/ctl-led/speaker/card${CardNumber}/attach:Master Playback Switch"
        ]                                                                           
}  
```

Reply:
probably best to let @perexg comment, I don't remember the nuance between BootSequence and FixedBootSequence and whether the Volume always comes with a Switch.